www/_template/reference/configuration.md

```diff
@@ -5,110 +5,290 @@ description: The Snowpack configuration API reference.
 ---
 
 ```js
-// Example Configuration File
+// Example: snowpack.config.js
 module.exports = {
   plugins: [
     /* ... */
   ],
-  installOptions: {
-    /* ... */
-  },
-  devOptions: {
-    /* ... */
-  },
-  buildOptions: {
-    /* ... */
-  },
-  mount: {
-    /* ... */
-  },
-  alias: {
+};
+```
+
+```js
+// Example: snowpack.config.js (ESM)
+// This is ESM-format config file. to enable
+// Add "type": "module" in your package.json
+export **Default** {
```

Comment:
This isn't formatted properly in a code block: https://www.snowpack.dev/reference/configuration

Also this doesn't work with the current release version of snowpack (2.18.5) with the same error as mentioned here: https://github.com/snowpackjs/snowpack/pull/968#issuecomment-683524633

Is this supposed to document the snowpack 3 behavior?